app/src/main/java/com/belatrixsf/allstars/adapters/EmployeeListAdapter.java

```diff
@@ -32,37 +38,38 @@ public EmployeeListAdapter(Context context, List<Employee> employeeList) {
 
     @Override
     public EmployeeViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
-
         View layoutView = LayoutInflater.from(parent.getContext()).inflate(R.layout.item_contact, parent, false);
-        EmployeeViewHolder rcv = new EmployeeViewHolder(layoutView);
-        return rcv;
+        return new EmployeeViewHolder(layoutView);
     }
 
     @Override
     public void onBindViewHolder(EmployeeViewHolder holder, int position) {
-        Employee emp = employeeList.get(position);
-
-        String fullName = emp.getFullName();
-        String letter;
-        if (null != fullName && !fullName.isEmpty()){
-            letter = String.valueOf(fullName.charAt(0));
-        }else{
-            letter = "#";
-        }
+        Employee employee = employeeList.get(position);
+        String fullName = employee.getFullName();
+        String currentFirstLetter = ( fullName != null && !fullName.isEmpty() ) ? String.valueOf(fullName.charAt(0)).toUpperCase() : "#";
```

Comment:
Also i would recommend to use static variables for 0 and #. something like
public static final String NUMERIC_SYMBOL = "#";